File: 프로그래머스/0/120819. 아이스 아메리카노/아이스 아메리카노.js
function solution(money) {
    let coffee = 5500
    let cupCount = Math.floor(money/coffee);
    let remainMoney = money%coffee;
    var answer = [cupCount,remainMoney];
    return answer;
}


 
// 머쓱이가 최대로 마실 수 있는 아메리카노의 잔 수와
 
// 남는 돈을 순서대로 담은 배열을 return 하도록
// solution 함수를 완성해보세요.

File: 프로그래머스/0/120819. 아이스 아메리카노/아이스 아메리카노.py
def solution(money):
    answer = []
    if money >= 5500:
        answer.append(money // 5500)          # 살 수 있는 커피 잔 수
        answer.append(money % 5500)           # 남는 잔돈
    else:
        answer = [0, money]                   # 커피 못 사면 잔돈 그대로
    return answer